Feature / Create & Style Get-start Section

liemgumball opened the pull request:
- Used import CSS files into index.css file
- Created Feature and Get-start section followed BEM naming convention
![image](https://github.com/liemgumball/html-css-training/assets/92854999/a449eeab-ed00-402a-b79a-f5972f772145)
- Applied Flexbox
![image](https://github.com/liemgumball/html-css-training/assets/92854999/0723c15c-8419-4cc3-b086-e5ec36209443)
- Added hover animation for buttons and no action
- Added images into assets/images folder

huyenpham-agilityio commented on practice-1/src/assets/css/get-start_section.css:
```diff
@@ -0,0 +1,72 @@
+.get-start-section {
```
file name not good

liemgumball replied:
how should I name this file?

liemgumball replied:
I changed it into get-start.css

huyenpham-agilityio commented on practice-1/src/assets/css/get-start_section.css:
```diff
@@ -0,0 +1,72 @@
+    top: 0;
+    left: 0;
+    width: 660px;
+    z-index: -99;
```
not good

huyenpham-agilityio commented on practice-1/src/assets/css/get-start_section.css:
```diff
@@ -0,0 +1,72 @@
+.get-start-section {
+    position: relative;
+    top:auto;
```
code style

huyenpham-agilityio commented on practice-1/src/assets/css/get-start_section.css:
```diff
@@ -0,0 +1,72 @@
+    z-index: -99;
+}
+
+.get-start__image--destinations {
```
destinations is modifier in BEM?

huyenpham-agilityio commented on practice-1/src/assets/css/get-start_section.css:
```diff
@@ -0,0 +1,72 @@
+    color: var(--font-color);
+}
+
+#icon--send {
+    top: 300px;
+    left: 50px;
+}
+
+#icon--add {
+    left: 400px;
+    bottom: -5px;
+}
+
+#icon--location {
+    bottom: 200px;
+    right: 0;
+}
```
avoid style for id please

huyenpham-agilityio commented on practice-1/src/assets/css/global.css:
```diff
@@ -0,0 +1,70 @@
+    font-size: var(--font-size-small);
+}
+
+.text--smaller {
+    font-size: var(--font-size-smaller);
```
use `smaller` is not good. Think about other name like tiny

huyenpham-agilityio commented on practice-1/src/assets/css/index.css:
```diff
@@ -15,28 +17,48 @@ html {
   --font-size-small: 18px;
   --font-size-normal: 24px;
   --font-size-big: 35px;
   --font-size-bigger: 44px;
```
bigger is not good too, large is a good option

huyenpham-agilityio commented on practice-1/src/assets/css/index.css:
```diff
@@ -15,28 +17,48 @@ html {
   font-family: 'Circular Std Medium', Arial, sans-serif;
+}
+
+#content {
```
avoid style for id

huyenpham-agilityio commented on practice-1/src/assets/images/OBJECTS1.svg:
this is removed in design

huyenpham-agilityio commented on practice-1/src/assets/images/Top destination.png:
image name

huyenpham-agilityio commented on practice-1/src/index.html:
```diff
@@ -3,8 +3,10 @@
   <head>
     <title>Travlog</title>
     <link rel="stylesheet" href="../src/assets/css/index.css" />
+    <link rel="stylesheet" href="../src/assets/css/global.css" />
     <link rel="stylesheet" href="../src/assets/css/header.css" />
     <link rel="stylesheet" href="../src/assets/css/footer.css" />
+    <link rel="stylesheet" href="../src/assets/css/get-start_section.css" />
```
import all css file in one file then add that file here

liemgumball commented:
I have fix all the comments, please review it @huyenpham-agilityio:

- Added the reset.css and imported into index.css
- Renamed file into get-start.css
- Renamed variables

huyenpham-agilityio commented on practice-1/src/assets/css/index.css:
```diff
@@ -1,42 +1,72 @@
+@import url(./footer.css);
+@import url(./get-start.css);
+
 html {
```
move out all css to other file, this file just only for importing

huyenpham-agilityio commented on practice-1/src/assets/images/Top_destination.png:
should be top-destination or topDestination
